### PR TITLE
feat(components/forms): tokenize field group styles

### PR DESCRIPTION
**Pull request by Blackbaud-ErikaMcVey**

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/2863/storybook/)

Component Storybooks:

- [action-bars-storybook](https://blackbaud.github.io/skyux-pr-preview/2863/storybooks/action-bars-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/2863/storybooks/ag-grid-storybook/)
- [angular-tree-component-storybook](https://blackbaud.github.io/skyux-pr-preview/2863/storybooks/angular-tree-component-storybook/)
- [avatar-storybook](https://blackbaud.github.io/skyux-pr-preview/2863/storybooks/avatar-storybook/)
- [colorpicker-storybook](https://blackbaud.github.io/skyux-pr-preview/2863/storybooks/colorpicker-storybook/)
- [data-manager-storybook](https://blackbaud.github.io/skyux-pr-preview/2863/storybooks/data-manager-storybook/)
- [datetime-storybook](https://blackbaud.github.io/skyux-pr-preview/2863/storybooks/datetime-storybook/)
- [errors-storybook](https://blackbaud.github.io/skyux-pr-preview/2863/storybooks/errors-storybook/)
- [forms-storybook](https://blackbaud.github.io/skyux-pr-preview/2863/storybooks/forms-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/2863/storybooks/indicators-storybook/)
- [inline-form-storybook](https://blackbaud.github.io/skyux-pr-preview/2863/storybooks/inline-form-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/2863/storybooks/layout-storybook/)
- [lists-storybook](https://blackbaud.github.io/skyux-pr-preview/2863/storybooks/lists-storybook/)
- [lookup-storybook](https://blackbaud.github.io/skyux-pr-preview/2863/storybooks/lookup-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/2863/storybooks/pages-storybook/)
- [phone-field-storybook](https://blackbaud.github.io/skyux-pr-preview/2863/storybooks/phone-field-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/2863/storybooks/split-view-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/2863/storybooks/tabs-storybook/)
- [text-editor-storybook](https://blackbaud.github.io/skyux-pr-preview/2863/storybooks/text-editor-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/2863/storybooks/theme-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/2863/code-examples/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/2863/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/2863/playground/)

File: apps/playground/src/app/components/layout/box/box.component.html [hunk 1]
```diff
@@ -133,3 +133,9 @@ <h2 class="sky-font-heading-2">Box 2 header</h2>
 </div>
 
 <ng-template #helpContent>Help content</ng-template>
+
+<div class="grandparent">
```
Comment (Blackbaud-TrevorBurch) [inline apps/playground/src/app/components/layout/box/box.component.html]: ?????

File: apps/playground/src/app/components/layout/box/box.component.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
+@mixin bg-color {
```
Comment (Blackbaud-TrevorBurch) [inline apps/playground/src/app/components/layout/box/box.component.scss]: ?????

File: apps/playground/src/app/components/layout/box/box.component.ts [hunk 3]
```diff
@@ -11,6 +11,7 @@ import { Component } from '@angular/core';
       }
     `,
   ],
+  styleUrl: './box.component.scss',
```
Comment (Blackbaud-TrevorBurch) [inline apps/playground/src/app/components/layout/box/box.component.ts]: ????

**Review by Blackbaud-TrevorBurch — APPROVED** [review]

**Pull request merged**